Example/AppDelegate.swift

```diff
@@ -1,23 +1,32 @@
 import Cocoa
 import Preferences
 
+extension NSToolbarItem.Identifier {
```

Comment:
I think we should use our own namespace for the identifiers instead of overloading `NSToolbarItem`. The identifier is for the whole preference pan, not just the toolbar item.

Reply:
Having our own namespace also lets us shorten the identifier from `preferenceGeneral` to just `general`.